etc/kayobe/ansible/opensearch_configure.yml

```
@@ -0,0 +1,7 @@
+---
+
+- hosts: localhost
+  tasks:
+    - include_role:
+        name: opensearch
+      when: kolla_enable_central_logging | bool
```

Not all deployments will be using opensearch.
```suggestion
      when: kolla_enable_opensearch | bool
```

---

Thanks - I'm fine with changing it. I'd assumed that anyone using this config would be deploying the reference monitoring stack.